modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc

```diff
@@ -5920,6 +5920,132 @@ Total database accesses: 5, total allocated memory: 1488
+| |\                          +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| | +Filter                   |  4 | stefan.name = $autostring_1                                                                      |              1 |    1 |      28 |                |                        |           |                     |
+| | |                         +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| | +NodeByLabelScan          |  5 | stefan:Person                                                                                    |             14 |   14 |      15 |            392 |                    1/0 |     0.482 | Fused in Pipeline 1 |
+| |                           +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| +Filter                     |  6 | chris.name = $autostring_0                                                                       |              1 |    1 |      28 |                |                        |           |                     |
+| |                           +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| +NodeByLabelScan            |  7 | chris:Person                                                                                     |             14 |   14 |      15 |            376 |                    3/0 |     2.092 | Fused in Pipeline 0 |
++-----------------------------+----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+
+Total database accesses: 125, total allocated memory: 24349
+
+----
+
+======
+
+
+[[query-plan-stateful-shortest-path-all]]
+== StatefulShortestPath(All)
+// StatefulShortestPath(All)
+
+The `StatefulShortestPath(All)` operator finds shortest paths from a previously matched node variable to an endpoint that was not previously matched.
+It uses a breath first search algorithm.
+
+
+.StatefulShortestPath(All)
+======
+
+.Query
+[source, cypher]
+----
+PROFILE
+MATCH (chris {name: 'Chris'})
+CALL {
+  WITH chris
+  MATCH
+    p = ALL SHORTEST (chris)(()-[]-()-[]-()){1,}(location:Location)
+  RETURN length(p) AS pathLength, location.name AS locationName
+}
```

Comment:
Why the subquery?

Reply:
I thought that it would make it more clear by explicitly saying that the query should start by matching `chris`. Also to make it more in line with the example for `INTO`. 
However, the example for `INTO` now doesn't need a `CALL` subquery due to the new `UNIQUE` index. Therefore we can remove the `CALL` subqueries in both cases now